src/main/webapp/app/exercises/shared/result/updating-result.component.ts

```diff
@@ -63,7 +59,6 @@ export class UpdatingResultComponent implements OnChanges, OnDestroy {
             const latestResult = this.participation.results && this.participation.results.find(({ rated }) => this.showUngradedResults || rated === true);
             // Make sure that the participation result is connected to the newest result.
             this.result = latestResult ? { ...latestResult, participation: this.participation } : undefined;
-            // TODO: how to determine that here?
             this.missingResultInfo = undefined;
```

Comment:
Consider removing this line and setting a default value. Refer to the other thread regarding the value itself.